src/Components/ShareImageModal/RegionsSelect.tsx

```diff
@@ -95,9 +100,11 @@ const RegionsSelect = ({
     if (selected.includes(selection)) {
       nextSelected = selected.filter((region) => region !== selection);
       setSelected(nextSelected);
+      setIsOpen(false);
```

Comment:
We discussed this offline, just adding a reminder here so I remember to double check myself that this was removed (multiple select shouldn't close open selecting an option).